database/localdb/localdbtest/test.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package localdbtest;
+
+message TestItem {
```

Comment:
Is this meant to be used by other packages outside of database/localdb? If not, consider putting it in database/localdb/internal.